core/src/main/scala/org/apache/spark/sql/TiContext.scala

```diff
@@ -68,6 +68,8 @@ class TiContext(val sparkSession: SparkSession) extends Serializable with Loggin
   clientSession.injectCallBackFunc(CacheInvalidateListener.getInstance())
   val meta: MetaManager = new MetaManager(clientSession.getCatalog)
   val debug: DebugTool = new DebugTool
+  val autoLoad: Boolean =
+    conf.getBoolean(TiConfigConst.ENABLE_AUTO_LOAD_STATISTICS, defaultValue = true)
```

Comment:
Shall we use `true` as the default value?

Reply:
Yes. For the former version, `true` is the default value.
And for TiDB, you can not disable statistics. https://docs.pingcap.com/tidb/dev/statistics